src/crypto/index.ts

```diff
@@ -3115,7 +3115,8 @@ export class Crypto extends TypedEventEmitter<CryptoEvent, CryptoEventHandlerMap
                 this.secretStorage.onRequestReceived(event);
             } else if (event.getType() === "m.secret.send") {
                 this.secretStorage.onSecretReceived(event);
-            } else if (event.getType() === "org.matrix.room_key.withheld") {
+            } else if (event.getType() === "m.room_key.withheld"
+                || event.getType() === "org.matrix.room_key.withheld") {
```

Comment:
These sorts of checks get easier with a `NamespacedValue`, but considering it'd be 1 line of code to help 2 other lines, the value is questionable.